src/main/java/MockClass.java

```diff
@@ -1,8 +1,25 @@
 
 public class MockClass {
 
+    /**
+     * cioc
+     * @param seed
+     * @return
+     */
     public int doSomething(int seed) {
 
         return seed + 5;
     }
+
+    public static void main(String args[]) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move the array designator from the variable to the type. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonarqube.tiolktrack.byznass.com:9000/coding_rules#rule_key=squid%3AS1197)